packages/react-core/src/components/LabelGroup/examples/LabelGroupEditableAdd.tsx

```diff
@@ -35,7 +40,7 @@ export const LabelGroupEditableAdd: React.FunctionComponent = () => {
         props: {
           isEditable: true,
           editableProps: {
-            'aria-label': 'label editable text'
+            'aria-label': `Editable label with text New Label`
```

Comment:
Not actually related to this line, but could we also update the aria-label for "Label 3" on line 14? Currently it still says "Label editable text" rather than following the verbiage you've updated elsewhere